test/factories/billing_expenses.rb

```diff
@@ -1,7 +0,0 @@
-FactoryGirl.define do
-  factory :billing_expense do
-    volunteer
-    user
-    amount 50
-  end
-end
```

Comment:
Why not keep the factory?

Reply:
Because it was a mere nightmare to get that working and passing FactoryGirl validation.
Thats why i killed it.

Reply:
It was not working and its multiple relations caused to either not beeing created for the test or not passing the FactoryGirl validation. If Factories are ment to make ones life easier, this one does certenly not meet that requirement. So it should be dropped.

Reply:
Fair enough!